### PR TITLE
Frio: restyle login page episode II • Attack of the Clones

### PR DESCRIPTION
**Pull request by fabrixxm**
followup of #4232 

- `mod/home.php` now uses `home.tpl` instead of generating the html  full in-code. Not fully tested, but generic template should give same result as previous code, so should work with other themes. Custom `home.html` is still loaded if exists in root, but now is included via template, so it can be a template itself and can use all variables defined for `home.tpl` (not really useful, but it comes for free and imho is nice)
- Fix text color in home with a home.html file, and fix layout
- Fix layout when "Community home" addon is enabled. As far as i can tell, it's the only addon using `"home_content"` hook, and it replaces home content completely, loading custom `home.html` by itself.  In `mod/home.php`and empty string is passed as input to the hook, and in `home.tpl` if and addon sets page content via `"home_content"` hook, this is used instead of header or `home.html` + login form. This cause to have the same behavior with "Community home", but the hook works a little differently. This should be documented.
- Small fix to layout of login form. Register link is now a button
- Add admin option to set login page background color. If both image and color are unset, default image is used. If image is unset, the color is visible through background gradient

Someone suggested to remove login button from topbar, but when using "Community home", no login form is displayed because Frio doesn't have the sidebar on home page. I'm for leaving it.
We could remove the register link, as is more prominent now in login form and anyway I think the icon is unclear, while the login one is more clear as suggests (to me) "enter", under which I expect to find either "login" or "register".

### CODE REVIEW

File: mod/home.php [hunk 1]
```diff
@@ -24,30 +24,38 @@ function home_init(App $a) {
...
 		unset($_SESSION['theme']);
 	}
 	if (x($_SESSION,'mobile-theme')) {
 		unset($_SESSION['mobile-theme']);
 	}
 
-	/// @TODO No absolute path used, maybe risky (security)
-	if (file_exists('home.html')) {
-		if (file_exists('home.css')) {
+	$customhome = False;
```
Comment (MrPetovan) [inline mod/home.php]: Standards: please use lowercase `false`.

File: view/theme/frio/css/style.css [hunk 2]
```diff
@@ -3008,8 +3008,17 @@ section .profile-match-wrapper {
 /** 
  * Login page
  */
+#login-submit-wrapper {
+	display: flex;
```
Comment (MrPetovan) [inline view/theme/frio/css/style.css]: Standards: Wrong indentation.

Comment (fabrixxm) [reply view/theme/frio/css/style.css]: ah! i missed one.. one! :-P

Comment (MrPetovan) [reply view/theme/frio/css/style.css]: 😄

**Pull request merged**